doc/CONFIGURATION.md

```diff
@@ -614,11 +616,12 @@ const node = await createLibp2p({
   ],
   keychain: {
     pass: 'notsafepassword123456789',
-    datastore: dsInstant,
-  }
+  },
+  datastore
```

Comment:
We now use `MemoryDatastore` from the https://www.npmjs.com/package/datastore-core package, could you update this accordingly?

Reply:
I changed it to `import { FsDatastore } from "datastore-fs";` as the purpose of including a datastore was to write to disc. MemoryDatastore only seem to be in memory. It that ok?

Reply:
Perhaps off topic, but, what is really the the difference here, "datastore-level" will use a datastore-interface wrapper around "classic-level", which in turn is a c++  lib with node bindings. "datastore-fs" instead relies on just javascript and Node APIs, functionality wise they are the same, but they might store things differently on disc?  Why is one preferred over another?

Reply:
yeap, makes sense!